engineio/asyncio_client.py

```diff
@@ -353,14 +357,15 @@ def _reset(self):
 
     async def _send_request(
             self, method, url, headers=None, body=None,
-            timeout=None):  # pragma: no cover
+            timeout=None, cert=None):  # pragma: no cover
         if self.http is None or self.http.closed:
             self.http = aiohttp.ClientSession()
         http_method = getattr(self.http, method.lower())
+        ssl_context = ssl.create_default_context(cafile=cert)
```

Comment:
It seems doing this every time a request is sent is not necessary, right? The context can be saved to a `this.ssl_context` member variable and reused.